ocrd_network/ocrd_network/__init__.py

```diff
@@ -0,0 +1,17 @@
+# This network package is supposed to contain all the packages and modules to realize the network architecture:
+# https://user-images.githubusercontent.com/7795705/203554094-62ce135a-b367-49ba-9960-ffe1b7d39b2c.jpg
+
+# For reference, currently:
+# 1. The WebAPI is available here:
+# https://github.com/OCR-D/ocrd-webapi-implementation
+# 2. The RabbitMQ Library (i.e., utils) is available here:
+# https://github.com/OCR-D/ocrd-webapi-implementation/tree/main/ocrd_webapi/rabbitmq
+# 3. Some potentially more useful code to be adopted for the Processing Server/Worker is available here:
+# https://github.com/OCR-D/core/pull/884
+# 4. The Mets Server discussion/implementation is available here:
+# https://github.com/OCR-D/core/pull/966
```

Comment:
Yes. A bit more clarification and update here: 
1. The `ocrd-webapi-implementation` repo implements the Discovery / Workflow / Workspace endpoints of the WebAPI currently. This Processing Server PR implements just the Processor endpoint of the WebAPI. Once we have this merged to core under ocrd-network, the other endpoints will be adapted to ocrd-network and then the `ocrd-webapi-implementation` repo can be archived for reference.
2. We have now integrated the RabbitMQ Library in the `ocrd-network` and using it.
3. I believe there is not any relevant code to be adopted from there (I will recheck that again) and it can be removed.
4. This is still up-to-date.

Reply:
@MehmedGIT  thanks for clarifying!

@tdoan2010 so is #884 (direct REST wrapper for processor CLI as `POST` – without queues and workers, only request backlog) still going to be (re-)implemented in some form, as an alternative?

Reply:
> @tdoan2010 so is #884 (direct REST wrapper for processor CLI as `POST` – without queues and workers, only request backlog) still going to be (re-)implemented in some form, as an alternative?

@tdoan2010 please clarify! (Even if this PR gets merged soon, the Processing Server / Queue model won't necessarily be used by all implementation projects. For those who want processor CLI more or less directly wrapped in REST to scale and distribute things, your #884 is still needed. Also, it could be re-used for the CLI interface across ocrd_all slim containers. – Otherwise, we would either have to give up the CLI interface and subscribe fully to processor workers, or not have a perspective for deployment via slim containers.)

Reply:
Ok, according to @tdoan2010 (via other communication), that's exactly the plan. So #884 will be rewritten later on. Closing here.